YoutubeDownloader/YoutubeDownloader.csproj

```
@@ -2,27 +2,45 @@
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
```

Should this be changed or does it need to be `WinExe` still?

```suggestion
    <OutputType>Exe</OutputType>
```

---

Yes, WinExe is still correct cross platform, if it was changed to Exe a console window would be started as well